packages/data-stores/package.json

```diff
@@ -38,8 +38,8 @@
 		"@wordpress/api-fetch": "^6.41.0",
 		"@wordpress/data-controls": "^3.13.0",
 		"@wordpress/deprecated": "^3.44.0",
-		"@wordpress/url": "^3.45.0",
 		"@wordpress/element": "^5.21.0",
+		"@wordpress/url": "^3.45.0",
```

Comment:
This came from merging trunk, it seems to be in alphabetical order now so I'm leaving it as is.